scripts/sk_cleanup_tenants/readme.md

```diff
@@ -0,0 +1,33 @@
+# Cleanup script for safekeeper
+
```

Comment:
Let's add a couple of lines actually saying what it does, i.e. removing tenant dirs on safekeepers of tenants which were removed in console.